pod/video/Encoding_video_model.py

```diff
@@ -236,7 +236,7 @@ def store_json_list_subtitle_files(self, info_video, video_to_encode):
             )
 
         for sub in list_subtitle_files:
-            if not check_file(list_subtitle_files[sub]):
+            if not check_file(list_subtitle_files[sub][1]):
```

Comment:
je suppose que la ligne précédente ne fonctionnait pas, mais tu peux mettre une petite explication ? :)

Reply:
TypeError: access: path should be string, bytes or os.PathLike, not list